CRM/Civicase/Service/CaseCategoryInstance.php

```diff
@@ -82,10 +82,19 @@ public function assignInstanceForExistingCaseCategories() {
    *   Instance ID.
    */
   public function createInstanceTypeFor($categoryValue, $instanceId) {
+    $instanceForProspect = civicrm_api3('CaseCategoryInstance', 'get', [
```

Comment:
Can we rename this variable to be generic and not necessarily refer to prospect.